docs/source/markdown/podman-create.1.md

```diff
@@ -850,6 +850,16 @@ If specified, the first argument refers to an exploded container on the file sys
 This is useful to run a container without requiring any image management, the rootfs
 of the container is assumed to be managed externally.
 
+  `Overlay Rootfs Mounts`
+
+   The `:O` flag tells Podman to mount the directory from the rootfs path as a
```

Comment:
Going back and forth over keeping as is or removing the 'a'....  your call.
```suggestion
   The `:O` flag tells Podman to mount the directory from the rootfs path as
```

Reply:
@TomSweeneyRedHat resolved in latest commit.